### PR TITLE
Closes #1251 - Reconcile the README.md TOC sections describing install between Mac, Linux, and Windows

### LINKED ISSUE #1251: Reconcile the README.md TOC sections describing install between Mac, Linux, and Windows
Reconcile the README.md TOC sections describing install between Mac, Linux, and Windows. Originally I thought we were missing some install stuff for MacOS but it turned out to be in a different order and you had to click through to see it. We need to clean this up especially with the potential addition of CentOS7 instructions. Maybe we should move this stuff into an INSTALL.md file.

### PR DESCRIPTION
**Pull request by Ethan-DeBandi99**
This PR (closes #1251):

I added an `INSTALL.md` file because I agreed with @mhmerrill and think that this helps with the organization and readability. The `README.md` now lists the prerequisites, but instead of detailing the installs out, it now links to `INSTALL.md`. TOC has been updated to reflect these changes.

The collapsible sections have been removed as they made it difficult to find certain elements. Additionally, all headings within the prereq install instructions is now linked to the TOC.

Linux section has been updated to indicate the instructions are specific to Ubuntu.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# Arkouda Prerequisite Install Guide
+
+This guide will walk you through the environment configuration for Arkouda to operate properly. Arkouda can be run on Linux, Windows, and MacOS.
+
+<a id="toc"></a>
+# Table of Contents
+
+1. [General Requirements](#genreqs)
+2. [Linux (Ubuntun)](#linux-unbuntu)
```
[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Corrected the typo

[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Corrected the typo

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+<a id="lu-python"></a> 
+### Python Environment - Anaconda (Ubuntu)<sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+As is the case with the MacOS install, it is highly recommended to [install Anaconda](https://docs.anaconda.com/anaconda/install/linux/) to provide a Python environment and manage Python dependencies:
+
+```bash
+ wget https://repo.anaconda.com/archive/Anaconda3-2020.07-Linux-x86_64.sh
+ sh Anaconda3-2020.07-Linux-x86_64.sh
+ source ~/.bashrc
+ 
+ # Install versioneer and other required python packages if they are not included in yoru anaconda install
```
[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Corrected typo

[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Corrected typo

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+ conda install versioneer
+ # OR
+ pip install versioneer
+ 
+ # Repeat for any missing pacakges using your package manager of choice (conda or pip)
```
[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Corrected typo

[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply INSTALL.md]: Corrected typo

**Review by mhmerrill — CHANGES_REQUESTED** [review]
i am ok with this, i want to see @hokiegeek2 's CentOS7 instructions added.

**Review by mhmerrill — APPROVED** [review]

**Pull request merged**